packages/nextjs-kit/src/components/contentWithImage.tsx

```diff
@@ -61,7 +64,7 @@ const ContentWithImage: React.FC<ContentProps> = ({
 			</div>
 
 			<div
-				className="ps-break-words ps-mt-12 ps-wp-content"
+				className={['ps-break-words ps-mt-12', contentClassName].join(' ')}
```

Comment:
Can we also inject it like so to avoid the `.join`?:

```suggestion
                                className={`ps-break-words ps-mt-12 ${contentClassName}`}
```

Reply:
The problem with that is that the rendered class when you don't pass a class name will have this `ps-break-words ps-mt-12 ` with the empty space and is not clean at all

What's usually done for this on the tailwind ecosystem is add a function to hide the join, I didn't add it to avoid premature abstraction let me know if that sounds better

Reply:
I just did it, because forgot to add a filter to remove the last injected class name if is undefined

Reply:
Ah I see, makes sense. Thanks!